owncloudApp/src/main/res/layout-land/account_setup.xml

```diff
@@ -257,9 +258,27 @@
 
             </LinearLayout>
 
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="7dp"
+                android:layout_marginBottom="7dp"
```

Comment:
Normally, we should try to avoid hardcoded dimens. Also, we should try to use consistent dimens. That's why we added some standard dimens here: https://github.com/owncloud/android/blob/master/owncloudApp/src/main/res/values/dims.xml#L21-L26 Try to use them when possible. 👍